tests/Controller/EnMarche/CitizenInitiativeControllerTest.php

```diff
@@ -38,6 +38,34 @@ public function testAnonymousUserCannotCreateCitizenInitiative()
         $this->assertClientIsRedirectedTo('/campus', $this->client);
     }
 
+    public function testAnonymousUserSeePartialName()
+    {
+        // Anonymous
+        $crawler = $this->client->request(Request::METHOD_GET, '/evenements');
+
+        $this->assertSame(2, $crawler->filter('.search__results__meta span:contains("Jacques P.")')->count());
+        $this->assertSame(0, $crawler->filter('.search__results__meta span:contains("Jacques Picard")')->count());
+
+        $crawler = $this->client->request(Request::METHOD_GET, '/initiative_citoyenne/'.LoadCitizenInitiativeData::CITIZEN_INITIATIVE_3_UUID.'/'.date('Y-m-d', strtotime('tomorrow')).'-apprenez-a-sauver-des-vies');
+
+        $this->assertSame('Organisé par Jacques P.', trim(preg_replace('/\s+/', ' ', $crawler->filter('.committee-event-organizer')->text())));
+    }
```

Comment:
You could ensure the full name never appears on the page by crawling all the content and asserting "Jacques Picard" does not appear at all, not only in the search results.

Reply:
The full name can appear for events.

Reply:
fair enough !